src/Creatable.js

```diff
@@ -151,6 +151,14 @@ const Creatable = React.createClass({
 		});
 	},
 
+	componentWillUpdate (nextProps, nextState) {
+		// Tricky: Async clears the select inputValue when deleting
+		// an option, but it doesn't fire onInputChange.
```

Comment:
It matches the rest of the file - wherever something is done to make it work nicely with Async there's a comment like this.